samcli/commands/package/core/command.py

```diff
@@ -0,0 +1,128 @@
+import click
+from click import Context, style
+from rich.table import Table
+
+from samcli.cli.core.command import CoreCommand
+from samcli.cli.row_modifiers import RowDefinition, ShowcaseRowModifier
+from samcli.commands.package.core.formatters import PackageCommandHelpTextFormatter
+from samcli.commands.package.core.options import OPTIONS_INFO
+from samcli.lib.utils.resources import resources_generator
+
+COL_SIZE_MODIFIER = 38
+
+
+class PackageCommand(CoreCommand):
```

Comment:
Quick explanation about what is this class used for would be useful if someone new to the codebase finds out this class.